pdlua_gfx.h

```diff
@@ -968,14 +1023,14 @@ static int stroke_ellipse(lua_State* L) {
 
     int line_width = luaL_checknumber(L, 5) * glist_getzoom(cnv);
 
-    const char* tags[] = { gfx->object_tag, register_drawing(gfx) };
+    const char* tags[] = { gfx->object_tag, register_drawing(gfx), gfx->current_layer_tag };
 
 #ifndef PURR_DATA
-    pdgui_vmess(0, "crr iiii ri rs rS", cnv, "create", "oval", x1, y1, x2, y2, "-width", line_width, "-outline", gfx->current_color, "-tags", 2, tags);
+    pdgui_vmess(0, "crr iiii ri rs rS", cnv, "create", "oval", x1, y1, x2, y2, "-width", line_width, "-outline", gfx->current_color, "-tags", 3, tags);
 #else // PURR_DATA
     int x0 = text_xpix((t_object*)obj, obj->canvas);
     int y0 = text_ypix((t_object*)obj, obj->canvas);
-    gui_vmess("gui_luagfx_stroke_ellipse", "xsssiiiii", cnv, tags[0], tags[1],
+    gui_vmess("gui_luagfx_stroke_ellipse", "xssssiiiii", cnv, tags[0], tags[1],
```

Comment:
Note to self: Tim encroached on PURR_DATA terrain here, need to fix that in post.